chat-dancer.iml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please do not put IDE files under version controll, this will annoy your team mates, as it will lead to merge conflicts.